build.psm1

```diff
@@ -407,12 +405,18 @@ Fix steps:
         return
     }
 
+    # Set $RuntimeRID as Win7-x64 or Win7-x86 so the package works on all supported Windows OS versions.
+    switch ($Runtime) {
+        'win-x64' { $RuntimeRID = 'win7-x64' }
+        'win-x86' { $RuntimeRID = 'win7-x86' }
+    }
+
     # set output options
     $OptionsArguments = @{
         CrossGen=$CrossGen
         Output=$Output
         FullCLR=$FullCLR
-        Runtime=$Runtime
+        Runtime=$RuntimeRID
```

Comment:
I'm confused by the reuse of the name `Runtime` - Now in Start-PSBuild it is used as "Generic" runtime. Here as minimal level runtime. I believe we should rename first or second.